src/client/app/containers/app.ts

```diff
@@ -73,12 +73,13 @@ export class AppComponent {
         return this.profile$
             .filter(profile => profile != null)
             .map(profile => {
-                if (!isNil(this.snippet.gistOwnerId)) {
-                    return this.snippet.gistOwnerId === profile.login;
+                if (isNil(this.snippet.gist)) {
```

Comment:
https://lodash.com/docs/#isNil

`isNil` covers both null and undefined. `isEmpty` sees if "value is an empty object, collection, map, or set", and strings specifically are considered empty with a length of 0. Not sure how it would handle undefined, which is what this condition is looking for.